extensions/amp-skimlinks/0.1/amp-skimlinks.js

```diff
@@ -159,7 +159,9 @@ export class AmpSkimlinks extends AMP.BaseElement {
   onClick_(eventData) {
     // The link was not monetizable or the link was replaced
     // by an other LinkRewriter.
-    if (eventData.linkRewriterId !== SKIMLINKS_REWRITER_ID) {
+    const doClickTracking = eventData.linkRewriterId !== SKIMLINKS_REWRITER_ID &&
```

Comment:
What about the `null` case we discussed? Maybe the linter doesn't let you do it?